nautobot/ipam/tables.py

```diff
@@ -120,10 +120,11 @@
 """
 
 VLAN_MEMBER_TAGGED = """
+{% load helpers %}
 {% if record.untagged_vlan_id == object.pk %}
-    <span class="text-danger"><i class="mdi mdi-close-thick"></i></span>
+    {{ false | render_boolean }}
 {% else %}
-    <span class="text-success"><i class="mdi mdi-check-bold"></i></span>
+    {{ true | render_boolean }}
 {% endif %}
 """
```

Comment:
This could arguably be simplified even further by removing this constant altogether and updating `VLANMembersTable` as follows:

```python
class VLANMembersTable(BaseTable):
    """
    Base table for Interface and VMInterface assignments
    """

    name = tables.LinkColumn(verbose_name="Interface")
    tagged = tables.Column(empty_values=(), orderable=False)

    def render_tagged(self, value, record):
        return render_boolean(record.untagged_vlan_id != self.context["object"].pk)
```

Reply:
Ah yes, will simplify this now.